integration-tests/utils/integration-helpers.js

```diff
@@ -1,10 +1,12 @@
 const _ = require('lodash/fp')
 const uuid = require('uuid')
 const {
-  click, clickable, dismissNotifications, fillIn, findText, gotoPage, input, signIntoTerra, waitForNoSpinners, navChild, noSpinnersAfter, navOptionNetworkIdle
+  click, clickable, dismissNotifications, fillIn, findText, gotoPage, input, signIntoTerra, waitForNoSpinners, navChild, noSpinnersAfter,
+  navOptionNetworkIdle
 } = require('./integration-utils')
 const { fetchLyle } = require('./lyle-utils')
 const { withUserToken } = require('../utils/terra-sa-utils')
+const { defaultLocation } = require('../../src/libs/runtime-utils')
```

Comment:
Ugh, I guess it isn't feasible to import from src in the integration tests. Feel free to go back to hard-coding us-central1.